<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>591</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="videoArea">
      <property name="text">
       <string>Video Area</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnStartStop">
      <property name="text">
       <string>Test Face Detections</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDataset">
      <property name="text">
       <string>Create Dataset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTraining">
      <property name="text">
       <string>Training</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnRecognition">
      <property name="text">
       <string>Recognition</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="infoArea">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>452</y>
     <width>581</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
